snap/snapcraft.yaml

```diff
@@ -0,0 +1,81 @@
+name: bisq-desktop
+base: core18
+version: 0.9.3
+summary: Peer-to-peer exchange to trade various curriencies for bitcoin
+description: |
+  Cross-platform desktop application that allows users to trade national currency (dollars, euros, etc) for bitcoin without relying on centralized exchanges
+confinement: strict
+grade: stable
+icon: icon.png
+
+apps:
+  bisq-desktop:
+    command: desktop-launch "$SNAP/opt/bisq/bin/bisq-desktop"
+    plugs:
+      - network
+      - network-bind
+      - desktop
+      - desktop-legacy
+      - x11
+      - wayland
+      - home
+    desktop: usr/share/applications/bisq.desktop
+    environment:
+      JAVA_HOME: "$SNAP/usr/lib/jvm/java-11-openjdk-amd64"
```

Comment:
Bisq should be built with Java 10. Java 11 isn't officially supported yet.

Reply:
I think `java-11-jdk` is Java 10 on Ubuntu. I have also learned that a Snap package can be created from the `.deb` if that would be better.

Reply:
Ubuntu 18.04 is still on Java 10 it seems. https://askubuntu.com/a/1037655/25776